packages/panda-preset/.gitignore

```diff
@@ -0,0 +1,3 @@
+node_modules/
+.DS_Store
+leather-styles/
```

Comment:
Maybe these can go in the global `.gitignore`, then we could avoid the nested one?

Reply:
Ok, thank you 👍